.travis.yml

```diff
@@ -15,14 +17,15 @@ matrix:
     - name: Unit tests
       env: DEBUG=1 DATABASE_URL=postgres:///django_ilmoitin
       services: [postgresql]
-      install: pip install -r requirements-test.txt
+      install: pip install -r requirements-test.txt -r requirements.txt
```

Comment:
you probably do not need to install `-r requirements.txt` if the packages are mentioned in `install_requires'